ibis/backends/polars/__init__.py

```diff
@@ -0,0 +1,194 @@
+from __future__ import annotations
+
+import importlib
+import re
+from functools import lru_cache
+from pathlib import Path
+from typing import Any, Mapping, MutableMapping
+
+import pandas as pd
+import pyarrow as pa
+
+import ibis.common.exceptions as com
+import ibis.expr.analysis as an
+import ibis.expr.operations as ops
+import ibis.expr.schema as sch
+import ibis.expr.types as ir
+from ibis.backends.base import BaseBackend
+from ibis.backends.datafusion.compiler import translate
```

Comment:
I would recommend against importing this, since it requires `datafusion` to be installed.

Reply:
Right, that's a leftover from the copy and paste. Will remove that.